include/npcomp/Dialect/Basicpy/IR/BasicpyOps.td

```diff
@@ -90,7 +90,65 @@ def CompareOperationAttr : StrEnumAttr<
 }
 
 //===----------------------------------------------------------------------===//
-// Ops
+// Ops that parallel Python bytecodes
+// Where possible and semantics allow, prefer to structure ops for compatibility
```

Comment:
add a mini-rationale? It's not obvious that copying a bytecode VM for a high-level dialect is the right choice.

Reply:
I just removed this and sorted them with the rest. My only rationale was to not diverge on naming if there was a similar construct already in that area.